[2022-02-17 13:49:55,467] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.task_a manual__2022-02-17T13:49:54.074055+00:00 [queued]>
[2022-02-17 13:49:55,479] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: my_dag.task_a manual__2022-02-17T13:49:54.074055+00:00 [queued]>
[2022-02-17 13:49:55,479] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-17 13:49:55,479] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-17 13:49:55,479] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-17 13:49:55,491] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_a> on 2022-02-17 13:49:54.074055+00:00
[2022-02-17 13:49:55,495] {standard_task_runner.py:52} INFO - Started process 3889 to run task
[2022-02-17 13:49:55,499] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'task_a', 'manual__2022-02-17T13:49:54.074055+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpj97_tcs4', '--error-file', '/tmp/tmpdjfastmg']
[2022-02-17 13:49:55,500] {standard_task_runner.py:77} INFO - Job 34: Subtask task_a
[2022-02-17 13:49:55,560] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.task_a manual__2022-02-17T13:49:54.074055+00:00 [running]> on host f8b49ae6ae82
[2022-02-17 13:49:55,619] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=task_a
AIRFLOW_CTX_EXECUTION_DATE=2022-02-17T13:49:54.074055+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-17T13:49:54.074055+00:00
[2022-02-17 13:49:55,620] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-17 13:49:55,621] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd -P']
[2022-02-17 13:49:55,630] {subprocess.py:85} INFO - Output:
[2022-02-17 13:49:55,632] {subprocess.py:89} INFO - /tmp/***tmpvm80_g72
[2022-02-17 13:49:55,632] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-17 13:49:55,657] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=task_a, execution_date=20220217T134954, start_date=20220217T134955, end_date=20220217T134955
[2022-02-17 13:49:55,674] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-17 13:49:55,703] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
